src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -128,8 +129,21 @@ public function getStyle($name)
     /**
      * {@inheritdoc}
      */
-    public function format($message)
+    public function format($message/*, $width = 0*/)
     {
+        if (func_num_args() >= 2) {
+            $width = func_get_arg(1);
+        } else {
+            if (__CLASS__ !== get_class($this)) {
+                $r = new \ReflectionMethod($this, __FUNCTION__);
+                if (__CLASS__ !== $r->getDeclaringClass()->getName()) {
+                    @trigger_error(sprintf('Method "%s()" will have a 2nd `$width = 0` argument in version 5.0. Not defining it is deprecated since 4.1.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
`... in version 5.0, not defining it is deprecated since Symfony 4.1`

Reply:
I don't understand why you want to force people to define the `$width`. Not setting it explicitly seems fine to me.